Merge community call 6 notes nad update merge.md (#607)
ethereum/pm: Merge/CommunityCall_Meeting_06.md

# Merge Community Call #6 Notes

### Meeting Date/Time: Friday 2022/08/12 at 14:00 UTC (10:00 ET)

### Meeting Duration: 1 hour

### [GitHub Agenda](https://github.com/ethereum/pm/issues/580)

### [Audio/Video of the meeting](https://www.youtube.com/watch?v=AZq1acbjaz4)

### Moderator: Trenton Van Epps

### Notes: Avishek Kumar

> —----------------------------

Reviewer: ```suggestion
---
```

## Summary 

> -  Plan to have another validator prep workshop  maybe next week or in two weeks as per Remy Roy 

Reviewer: ```suggestion
-  Plan to have another merge community call in September 
```

> —----------------------------

Reviewer: ```suggestion
---
```

> **Trent**[6:54](https://www.youtube.com/watch?v=AZq1acbjaz4&t=414s): Great job Tim, yeah I think you have covered everything with all the updates.

Reviewer: ```suggestion

**Trent**[6:54](https://www.youtube.com/watch?v=AZq1acbjaz4&t=414s): Great job Tim, yeah I think you have covered everything with all the updates.
```

**Trent** [49:13](https://www.youtube.com/watch?v=AZq1acbjaz4&t=2953s): One last question yeah i don't know if we can explain replay attacks but I am not gonna get into like how to financially profit from this  upgrade replay attack is when you send a transaction on one chain and it can be replayed on a chain that's similar but not exactly the same as the one that you sent the original transaction so aside from that yeah.

**Tim Beiko** [49:55](https://www.youtube.com/watch?v=AZq1acbjaz4&t=2995s): Okay yeah I think we got it.  Thanks everyone for showing up and yes you in about a month for the next one of these .

**Trent** [50:05](https://www.youtube.com/watch?v=AZq1acbjaz4&t=3005s): All right thank you again to the client devs and researchers who showed up and answered questions. I really appreciate you spending time on this so yeah we'll see everybody in a month or in the discord or elsewhere bye

----------------------------------------------------------------
## Attendance

* Trenton Van Epps  
* Thorsten Behrens
* Phil Ngo 
* Tim Beiko 
* Remy Roy
* Terence Tsao
* Marius VanDer Wijden
* Pooja Ranjan
* Parithosh
* Danny

> ## Next Meeting Date/Time : TBD

Reviewer: ```suggestion
## Next Meeting Date/Time : Sept 9 at 14:00 UTC (10:00 ET)
```

> 

Reviewer: ## Zoom chat
